[SRVCOM-1831] Serverless functions: update abstracts format (#46660)
openshift/openshift-docs: modules/serverless-invoking-go-functions-http.adoc

[id="serverless-invoking-go-functions-http_{context}"]
= Functions triggered by an HTTP request

> When an incoming HTTP request is received, your function is invoked with a standard Golang link:https://golang.org/pkg/context/[Context] as the first parameter, followed by the link:https://golang.org/pkg/net/http/#ResponseWriter[`http.ResponseWriter`] and link:https://golang.org/pkg/net/http/#Request[`http.Request`] parameters. You can use standard Golang techniques to access the request, and set a proper HTTP response of your function.

Reviewer: Nit
```suggestion
When an incoming HTTP request is received, your function is invoked with a standard Go link:https://golang.org/pkg/context/[Context] as the first parameter, followed by the link:https://golang.org/pkg/net/http/#ResponseWriter[`http.ResponseWriter`] and link:https://golang.org/pkg/net/http/#Request[`http.Request`] parameters. You can use standard Go techniques to access the request, and set a proper HTTP response for your function.
```
